server/router/common.ts

```diff
@@ -1,7 +1,7 @@
 import { Request, Response } from 'express';
 import { DocumentClient } from 'aws-sdk/clients/dynamodb';
 
-export const deleteByID =
+export const deleteByID = (
```

Comment:
Will this entire file be needed at all if we are using dynamoose? Maybe consider replacing the delete endpoints in another PR with their dynamoose equivalents?

Reply:
in my next PR i'm currently working on adding common functions like getByID, getAll, etc. and rewriting deleteByID using dynamoose, i just thought it would be a lot for this one